velox/expression/EvalCtx.cpp

```diff
@@ -231,16 +231,16 @@ void EvalCtx::setErrors(
   rows.applyToSelected([&](auto row) { setError(row, exceptionPtr); });
 }
 
-const VectorPtr& EvalCtx::getField(int32_t index) const {
```

Comment:
Why did you have to change the const-ness here (const_cast always makes me scared)